### PR TITLE
DOC Update README.md for CMS 5

### PR DESCRIPTION
**Pull request by GuySartorelli**
## Issue

- https://github.com/silverstripe/.github/issues/20

### CODE REVIEW

**Review by sabina-talipova — CHANGES_REQUESTED** [review]
Please, move the added section in `About this module` section before `Configuration` section, to be consistence with another module's README

**Comment by GuySartorelli** [comment]
That's not in scope for this issue - this issue is for updating readme content to be correct for CMS 5. The order of items in the readme is not relevant to that task.

There are far too many modules and far too many possible permutations to worry about that in this issue. It could take ages to validate that everything is consistent.

**Comment by sabina-talipova** [comment]
> That's not in scope for this issue - this issue is for updating readme content to be correct for CMS 5. The order of items in the readme is not relevant to that task.

I think, since it's a new section we can append it in "correct" section.

**Comment by GuySartorelli** [comment]
Neither "About this module" nor "Configuration" are new sections? Maybe I've misunderstood your request... can you please rephrase it to be more explicit?

**Comment by sabina-talipova** [comment]
> Neither "About this module" nor "Configuration" are new sections? Maybe I've misunderstood your request... can you please rephrase it to be more explicit?

You have added new `Installation` section. Could you, please, move `Installation` into `About this module` section before `Configuration`.

**Comment by GuySartorelli** [comment]
Ahhh, I see what you mean now. Though I don't think that's actually going to be more consistent than the current situation. All of the installation sections that I added were added as the first second-level heading. Which other module are you referring to that has the installation section as a third-level heading under some "about the module" heading?

**Comment by sabina-talipova** [comment]
If you have a look at these modules you will see that `Installation` section is after small `Overview` . 
https://github.com/creative-commoners/cwp-search/blob/e3f5bd22963745e5283b149ad9d21aa9c35654f4/README.md
https://github.com/creative-commoners/cwp-starter-theme/blob/d2dee3de3b0880d2bda3c3c65ccde4edcd97c2e4/README.md
https://github.com/silverstripe/silverstripe-admin/blob/1efa52486f0219b33b744d012399759ead041da8/README.md

**Comment by GuySartorelli** [comment]
Ahh okay. I don't think it really matters much but no worries I'll make it consistent with the way admin does it, i.e. installation will be a second-level heading underneath the "About this module" heading. This will also mean "Configuration" becomes a second-level heading.

**Comment by GuySartorelli** [comment]
Done. Since I was in here editing this one manually I also added correct code syntax highlighting.

**Review by sabina-talipova — APPROVED** [review]
LGFM
